x-pack/plugin/searchable-snapshots/src/test/java/org/elasticsearch/xpack/searchablesnapshots/cache/common/CacheFileTests.java

```diff
@@ -393,15 +393,7 @@ public void testFSyncFailure() throws Exception {
         }
     }
 
-    private static void assumeLinux64bitsOrWindows() {
-        assumeTrue(
-            "This test uses native methods implemented only for Windows & Linux 64bits",
-            Constants.WINDOWS || Constants.LINUX && Constants.JRE_IS_64BIT
-        );
-    }
-
     public void testCacheFileCreatedAsSparseFile() throws Exception {
-        assumeLinux64bitsOrWindows();
```

Comment:
I see you are removing this, even if the pre-allocate for Windows is a no-op; I think you are right, a quick glance at the test seems to indicate the assertions would be working on Windows too. Just to be safe, I suggest to make a final test run with `test-windows`.